package.json

```diff
@@ -114,7 +114,7 @@
     "query-string": "^6.2.0",
     "rc-tooltip": "^3.7.3",
     "react": "^16.8.0",
-    "react-autosuggest": "^9.4.3",
+    "react-autosuggest": "^10.0.2",
```

Comment:
Is it related?

Reply:
This was because the control was not working right and I wanted to ensure the problem would not be resolved by upgrading (it wasn't).  Since in general we support keeping our packages up-to-date, I left this in.